### PR TITLE
Fix slow descriptor regex some more

### PR DESCRIPTION
**Pull request by erikrose**
Replaces #601

### CODE REVIEW

File: dxr/plugins/descriptor/__init__.py [hunk 1]
```diff
@@ -57,7 +57,10 @@ def needles(self):
 class FileToIndex(dxr.indexers.FileToIndex):
     """Do lots of work to yield a description needle."""
 
-    comment_re = re.compile(r'(?:.*?/\*+)(?:\s*\*?\s*)(?P<description>.*?)(?:(?:\*+/)|(?:$))', flags=re.S)
+    # comment_re anchors at start of line and finds everything after /* until
+    # the first / in order to match entire javadoc-style comments where every
+    # subsequent line starts with a * (e.g. to find entire licenses).
```
Comment (pelmers) [inline dxr/plugins/descriptor/__init__.py]: Does this comment still apply?

**Comment by pelmers** [comment]
Looks perfect, other than the one comment! r+

**Comment by pelmers** [comment]
Also resolves https://bugzilla.mozilla.org/show_bug.cgi?id=1289411

File: dxr/plugins/descriptor/__init__.py (hunk 1 above)
Comment (erikrose) [inline dxr/plugins/descriptor/__init__.py]: Nope! Fixed.

**Pull request closed without merging**